### DISCUSSION TITLE
Valet not working on Chrome, but does work on Safari

### DISCUSSION THREAD

**Discussion by deplorableword** [laravel/valet#1484] [General]
Hello, running Sonoma 14.4, PHP 8.3. When I run valet links, I get 

```
+-------+-----+--------------------+----------------------------------------------+-------------+
| Site  | SSL | URL                | Path                                         | PHP Version |
+-------+-----+--------------------+----------------------------------------------+-------------+
| blah |  X  | https://blah.test | /Users/tom/Desktop/blahl | php@8.3     |
+-------+-----+--------------------+----------------------------------------------+-------------+
```

When I run valet status I get

```+--------------------------------------+----------+
| Check                                | Success? |
+--------------------------------------+----------+
| Is Valet fully installed?            | Yes      |
| Is Valet config valid?               | Yes      |
| Is Homebrew installed?               | Yes      |
| Is DnsMasq installed?                | Yes      |
| Is Dnsmasq running?                  | Yes      |
| Is Dnsmasq running as root?          | Yes      |
| Is Nginx installed?                  | Yes      |
| Is Nginx running?                    | Yes      |
| Is Nginx running as root?            | Yes      |
| Is PHP installed?                    | Yes      |
| Is linked PHP (php) running?         | Yes      |
| Is linked PHP (php) running as root? | Yes      |
| Is valet.sock present?               | Yes      |
+--------------------------------------+----------+
```

On Safari the site is found and works with SSL on Chrome I get 

```This site can’t be reachedCheck if there is a typo in sigil.test.
DNS_PROBE_FINISHED_NXDOMAIN
```

Any ideas on what I can do to troubleshoot this further? 

Thanks

**Comment by drbyte** [comment]
`DNS_PROBE_FINISHED_NXDOMAIN` is basically "I don't know (or can't look up) the IP address for that domain name" (basically, "DNS lookups are failing").

When it appears in all browsers on the same computer, it suggeststhe DNS service (Valet uses dnsmasq) is down/broken.

However ... when it appears in only a specific browser, it's usually because you've configured something in that browser to handle DNS differently than the rest of your Mac does. 
That'll be your browser's network or DNS settings. 
Sometimes it happens when you enable a VPN plugin, configured proxying, or you set up DNS-over-HTTPS ("secure DNS"), and those are interfering with normal DNS activity.
It can also happen after tinkering in Chrome's experimental-flags features.

Sometimes you can test this out and even "fix" it by creating a completely new browser (in your case, Chrome) profile and resetting all its settings to defaults. And you can reset all experimental flags as well.
